test/extended/util/test.go

```diff
@@ -221,6 +221,18 @@ func AnnotateTestSuite() {
 			node.SetText(node.Text() + " [Suite:k8s]")
 		}
 		node.SetText(node.Text() + labels)
+
+		skipTestsEnvVar, skipTestsEnvVarExists := os.LookupEnv("SKIP_TESTS")
+
+		if skipTestsEnvVarExists {
+			skipTests := strings.Split(skipTestsEnvVar, "\n")
```

Comment:
You should move the `os.LookupEnv` and `strings.Split` outside of the `ginkgo.WalkTests` loop so you're not reparsing it every time. Otherwise this looks good.